portal/models/user.py

```diff
@@ -469,6 +479,28 @@ def current_encounter(
             return initiate_encounter(self, auth_method=existing.auth_method)
         return existing
 
+    TOKEN_LEN = 4
```

Comment:
Prefer a more unique name, just in case we need to make this configurable...
```suggestion
    TOTP_TOKEN_LEN = 4
```